apps/web/pages/settings/teams/[id]/dsync.tsx

```diff
@@ -0,0 +1,9 @@
+import DirectorySyncTeamView from "@calcom/features/ee/dsync/page/team-dsync-view";
+
+import type { CalPageWrapper } from "@components/PageWrapper";
+import PageWrapper from "@components/PageWrapper";
+
+const Page = DirectorySyncTeamView as CalPageWrapper;
```

Comment:
how can I get to this page?

Reply:
Let's remove this page since we're not providing dsync for teams now